app/models/article.rb

```diff
@@ -1,6 +1,7 @@
 class Article < ApplicationRecord
 
   has_many :comments, dependent: :destroy
+  belongs_to :user, foreign_key: :user_id
```

Comment:
Could you have done `foreign_key: :user_id` when Rails migration could have done that better? Since this implementation does not reflect in the DB schema?

Reply:
Noted, thanks for the feedback